apps/video/index.html

```diff
@@ -65,59 +65,73 @@ <h1 id="thumbnails-number-selected"></h1>
         <!-- display this at startup while we create thumbnails -->
-        <div id="spinner-overlay" class="hidden"><div id="spinner"></div></div>
+        <div id="spinner-overlay" class="hidden" role="progress"
+             data-l10n-id="spinner-overlay"><div id="spinner"></div></div>
         <div id="video-container">
-          <video src="about:blank" id="player"></video>
+          <video src="about:blank" id="player" aria-hidden="true"></video>
         </div>
         <!-- video controls -->
         <div id="videoControls">
           <section role="region">
             <gaia-header id="player-header" action="back">
               <h1 id="video-title"></h1>
-              <button id="options" data-icon="more"></button>
+              <button id="options" data-icon="more"
+                      data-l10n-id="options-more"></button>
               <button id="picker-done" data-l10n-id="done">Done</button>
             </gaia-header>
-            <footer id="videoBar">
-              <div id="timeSlider">
-                <span id="elapsed-text"></span>
-                <div id="slider-wrapper">
-                  <div id="elapsedTime" class="progress"></div>
-                  <div id="bufferedTime" class="progress"></div>
-                  <div id="timeBackground" class="progress"></div>
-                  <button id="playHead"></button>
+            <div id="hide-controls" role="button"
+                 data-l10n-id="hide-controls-button"></div>
+            <div role="toolbar">
+              <footer id="videoBar" role="presentation">
+                <div id="timeSlider" role="slider">
+                  <span id="elapsed-text" aria-hidden="true"></span>
+                  <div id="slider-wrapper" aria-hidden="true">
+                    <div id="elapsedTime" class="progress"></div>
+                    <div id="bufferedTime" class="progress"></div>
+                    <div id="timeBackground" class="progress"></div>
+                    <button id="playHead"></button>
+                  </div>
+                  <span id="duration-text" aria-hidden="true"></span>
                 </div>
-                <span id="duration-text"></span>
-              </div>
-              <div id="fullscreen-button"></div>
-            </footer>
-            <footer id="videoControlBar">
-              <div id="videoToolBar">
-                <button id="seek-backward" class="player-controls-button" data-icon="skip-back"></button>
-                <button id="play" class="player-controls-button" data-icon="pause"></button>
-                <button id="seek-forward" class="player-controls-button" data-icon="skip-forward"></button>
-              </div>
-            </footer>
+                <div id="fullscreen-button" role="button"
+                     data-l10n-id="fullscreen-button"></div>
+              </footer>
+              <footer id="videoControlBar" role="presentation">
+                <div id="videoToolBar">
```

Comment:
Should videoToolBar have a l10n-id -- 'video-toolbar'? That id exists in the locales properties file...